modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the images provided in the client repository for this operating system are out of date.
+Bootstrapping new {centos} 6 clients using these packages will fail.
+This failure will not happen on {centos} 6 clients that are already installed and bootstrapped.
+
+If you need to bootstrap new {centos} 6 clients you can edit the existing repositories to reflect the correct URL.
+
+
+
+.Procedure: Troubleshooting Bootstrapping a New {centos} 6 Client
+. On the {centos} 6 client, at the command prompt, open the ``CentOS-Base.repo`` file, located in the ``/etc/yum/repos.d/`` directory.
+. Locate the ``baseurl`` line, pointing to ``mirrorlist.centos.org``.
+There may be more than one entry.
+For example:
++
+----
+baseurl=http://mirrorlist.centos.org/?release=6&arch=$basearch&repo=os
```

Comment:
I have to ask again: are you 100% sure this is `baseurl=`?

Usually at Yum, mirrors use `mirrorlist=`.

Reply:
I will test on my VM again today for you.

Reply:
This is what I have on my CentOS 6 machine:

![image](https://user-images.githubusercontent.com/3914967/109252816-250d5880-783a-11eb-962c-4e4151a1e898.png)

When I run `yum update` it is successful, although I've just now realised I'm getting a couple of warnings right before it loads the packages:

![image](https://user-images.githubusercontent.com/3914967/109252961-73baf280-783a-11eb-8717-27c7446dba55.png)

If I change the mirrorlist (just the first entry):

![image](https://user-images.githubusercontent.com/3914967/109253790-3fe0cc80-783c-11eb-8010-21fcbad3e150.png)

Now `yum update` still works, but now I get this error instead:

![image](https://user-images.githubusercontent.com/3914967/109253698-0f992e00-783c-11eb-8efd-074c7c87ebf0.png)

If I comment out the `baseurl` entry:

![image](https://user-images.githubusercontent.com/3914967/109253890-80404a80-783c-11eb-897d-552553055e52.png)

Now it stops working altogether:

![image](https://user-images.githubusercontent.com/3914967/109253870-74ed1f00-783c-11eb-872c-e316ea34e8ca.png)

If I revert to the old mirrorlist URL, and keep the baseurl commented:

![image](https://user-images.githubusercontent.com/3914967/109253952-b1207f80-783c-11eb-8aad-e08ec316a56a.png)

It fails completely again:

![image](https://user-images.githubusercontent.com/3914967/109254009-d1e8d500-783c-11eb-851e-24454367b72b.png)

My best guess is that it's looking for the URL in the mirrorlist, failing, and falling back to the baseurl, which is succeeding. You might have a better understanding though.

HTH.

Reply:
As I guessed, the VM has `mirrorlist` enabled, so the PR needs to be changed.

What you need, as a user is:

1. Comment the `mirrorlist=` line.
2. Make sure you have a `baseurl=` line with the vault repository that is "equivalent" to the one you had at `mirrorlist=`

And that, for all repositories.

Try the procedure folllowing that instruction and you will see it works just fine.

Reply:
Yep, that's got it. I'll update the content.

![image](https://user-images.githubusercontent.com/3914967/109451252-c5f94f00-7a98-11eb-8328-90bb3b0c64e5.png)

Reply:
I still see `baseurl=http://mirrorlist.centos.org/?release=6&arch=$basearch&repo=os` as the string to be commented? Wha am I missing? Shouldn't it mention `mirrorlist=` instead?

Reply:
Where do you see that? L81 says to comment out `mirrorlist` :point_down:

Reply:
Line 71.

Reply:
I still see this:

```
.Procedure: Troubleshooting Bootstrapping a New {centos} 6 Client
. On the {centos} 6 client, at the command prompt, open the ``CentOS-Base.repo`` file, located in the ``/etc/yum.repos.d/`` directory.
. Locate the ``baseurl`` line, pointing to ``mirrorlist.centos.org``.
There may be more than one entry.
For example:
+
----
baseurl=http://mirrorlist.centos.org/?release=6&arch=$basearch&repo=os
----
```
AFAIK such line doesn't exist, because it should start with `mirrorlist=` and it's what you need to comment (as you tell the user to do at line 81).

And also I don't think you need to edit ". Edit each ``baseurl`` line to point to a ``vault.centos.org`` URL." :-?

If the user does that, the consequence is enabling all the 6.X repositories as well.

The instructions should be rather something like this:

> This is an example to replace the `os` repository:
> 1. Find the line `mirrorlist=http://mirrorlist.centos.org/?release=6&arch=$basearch&repo=os`
> 2. Comment the line
> 3. Add a new line pointing to the `os` repository at `vault.centos.org`: `baseurl=https://vault.centos.org/centos/6/os/$basearch/`
> Repeat for all other repositories you are using from `mirrorlist.centos.org`

NOTE: Test the procedure I described, just in case, but I am 99.99% sure it will work.

Reply:
Both of our procedures end up with the same result, I've tidied mine up to make it a little more logical, since there was obviously confusion in the way I wrote it originally. Note that L71 doesn't say to comment anything out, it just says to *locate* the line to work on :wink: Also, there's no need to add a new baseurl entry, they're all already there on a standard install.